src/app/pages/elements/form/FormDemo.ts

```diff
@@ -387,7 +387,7 @@ export class FormDemoComponent {
       { token: 'TOKEN', military: true },
       {
         customfield: {
-          customControl: CustomDemoComponent,
+          customControl: 'custom-demo-component',
```

Comment:
Can this be renamed to `template` just to align with tables? They pass `template` to the column config to specify a custom cell.

Reply:
sure

Reply:
done